### ISSUE TITLE
Encourage x_contributors

### ISSUE THREAD

**Issue by szabgab** [book/CPANio#7]
There is a field called x_contributors  in the META files that is supposed to list all the contributors of a module http://perlmaven.com/how-to-add-list-of-contributors-to-the-cpan-meta-files
I think it is not used by a lot of people but by creating a metric of "number of modules where user N is listed in the x_contributors field (and even linking to the modules?) could help promoting the field and getting more people to actually contribute to modules. Even when they are not the maintainers.

**Comment by book** [comment]
What are the differents ways an author can contribute to a distribution (from a PAUSE/CPAN point of view)?

I count three:
* author (published the first release)
* maintainer (published some subsequent release)
* contributor (listed in x_contributors)

I would assume that each group contains the next one (an author is also a maintainer and a contributor, and a maintainer is also a contributor).

So we could have three boards, with a prominent explanation of what it means to be a contributor.

**Comment by neilbowers** [comment]
we could also consider the following to be "contribution"

 * submitting a bug, especially if it has a patch attached
 * a feature request / suggestion, which gets implemented

**Comment by szabgab** [comment]
* Submit an example for usage with (or even without?) explanation.

**Comment by book** [comment]
I was of course considering things that can actually be measured.

Submitting bugs (both RT tickets and github issues) can be measured I guess, provided we have a mapping for email address/CPAN authors (that's actually also required for x_contributors). I guess that role could be named "reporter".

The main difference with the first three I mentionned, is that "reporting" happens outside the control of distribution authors (x_contributors being a META field).

**Comment by book** [comment]
Another comment about "contributor" is that it can go away (if one's name is removed from the field during a later release). So, for the sake of correctness, I would look at all releases of all distributions.

**Comment by szabgab** [comment]
I envisioned the x_contributor as a field that list anyone who contributed at any given time to the project. Regardless of the amount and 'recentness' of contribution.

**Comment by book** [comment]
@szabgab yes, absolutely. I'm just saying that because the field can be updated (added to and deleted from) with every release of a distribution, it's possible that for some reason someone is removed.

We can decide to either trust the latest version always, or keep track of everyone ever listed in a distribution's x_contributor, no matter what the latest version says.

**Comment by karenetheridge** [comment]
I would just look at the most recent release.  Generally we presume that if something was removed from a dist, it is supposed to no longer be in effect.
